paddle/scripts/paddle_build.sh

```diff
@@ -204,6 +204,7 @@ function cmake_gen() {
         -DWITH_INFERENCE_API_TEST=${WITH_INFERENCE_API_TEST:-ON}
```

Comment:
There is a "distibuted_flag" in this file which could be changed to distributed_flag.